recipe/meta.yaml

```diff
@@ -1,52 +1,38 @@
-{% set name = "SDL2" %}
 {% set version = "2.0.8" %}
-{% set sha256 = "edc77c57308661d576e843344d8638e025a7818bff73f8fbfab09c3c5fd092ec" %}
 
 package:
-  name: {{ name|lower }}
+  name: sdl2
   version: {{ version }}
 
 source:
-  fn:  {{ name }}-{{ version }}.tar.gz
-  url:  https://www.libsdl.org/release/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url:  https://www.libsdl.org/release/SDL2-{{ version }}.tar.gz
+  sha256: edc77c57308661d576e843344d8638e025a7818bff73f8fbfab09c3c5fd092ec
 
 build:
-  number: 1
-  features:
-    - vc9  # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py35]
+  number: 1001
 
 requirements:
   build:
-    # Add python for feature activation
-    - python  # [win]
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py35]
     - cmake  # [win]
-    - autoconf  # [unix]
+    - autoconf  # [not win]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
     - xorg-libx11  # [linux]
-  run:
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py35]
+    - xorg-libxext  # [linux]
```

Comment:
I think these need to be in `run` as well.

Reply:
All? So just adding the section

```
run:
    - vc 9  # [win and py27]
    - vc 10  # [win and py34]
    - vc 14  # [win and py35]
    - xorg-libx11  # [linux]
    - xorg-libxext  # [linux]
```
should do the trick? And don't the vc deps need to be available in the build (or host) section as well?

Reply:
The `vc` stuff is no longer needed and in fact not recommended for recipes. This is because `conda-build` adds the `vc` constraint for us under the hood.

ref: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/6ea94a12393e64af171ee0936d71f3ddd8de4667/recipe/conda_build_config.yaml#L471-L474

Reply:
To clarify my original comment, was referring only to the `xorg-*` packages. HTH

Reply:
Ah, I thought this was meant to fix the windows builds, which are the only ones still failing :smile: I'll undo my last additions. It's still weird though that the win build voor the higher VC version fails while the VC9 versions builds without problems.

Reply:
Sorry about that.

Yeah it's definitely unusual.

Maybe a [missing library issue]( https://stackoverflow.com/a/36178263 )?

Reply:
Thanks for that! I'll try the suggestion in your linked post of adding vcruntime.lib as an explicit dependency soon, when I have the time.